src/actions.js

```diff
@@ -60,21 +62,50 @@ export function createSUSIMessage(createdMessage, currentThreadID) {
     timeout: 3000,
     async: false,
     success: function (response) {
-       receivedMessage.text = response.answers[0].actions[0].expression;
-       receivedMessage.response = response;
-        let message =  ChatMessageUtils.getSUSIMessageData(
-          receivedMessage, currentThreadID
-        );
-
-        ChatAppDispatcher.dispatch({
-          type: ActionTypes.CREATE_SUSI_MESSAGE,
-          message
+        receivedMessage.text = response.answers[0].actions[0].expression;
+        receivedMessage.response = response;
+        let actions = [];
+        response.answers[0].actions.forEach((actionobj) =>{
+          actions.push(actionobj.type);
         });
+        receivedMessage.actions = actions;
+        if (actions.indexOf('websearch')>=0) {
+          let actionIndex = actions.indexOf('websearch');
+          let query = response.answers[0].actions[actionIndex].query;
+          $.ajax({
+            url: 'http://api.duckduckgo.com/?format=json&q='+query,
+            dataType: 'jsonp',
+            crossDomain: true,
+            timeout: 3000,
+            async: false,
+            success: function (data) {
+                receivedMessage.websearchresults = data.RelatedTopics;
```

Comment:
Can we have the Abstract to be the first card with the image. And then the RelatedTopics? @uday96